 CONTRIBUTING.md

```diff
@@ -0,0 +1,76 @@
+# Contribution guide
+
+welcome to Litestar! We're excited that you're interested in contributing to Litestar.dev. Before you get started,
+please read the following guidelines. If you have any questions, feel free to ask in
+the [Litestar Discord server](https://discord.gg/X3FJqy8d2j). We're happy to help!
+
+## Setting up the environment
+
+1. Install [Poetry](https://python-poetry.org/) - Run `curl -sSL https://install.python-poetry.org | python3 - ` to
+   install poetry.
+2. Install [Node.js](https://nodejs.org/en) - `https://nodejs.org/en`.
+3. Install [Pre-commit](https://pre-commit.com/) - Run `pip install pre-commit` to install pre-commit hooks.
+4. Run `poetry install` and run `pip install sphinx-autobuild`.
```

Comment:
```suggestion
4. Run `make install` to install Python and Node dependencies.
```

`sphinx-autobuild` is included in the poetry dependencies.

Reply:
anas have faced this issue, we have to run `pip install sphinx-autobuild` at least once else it throws an error.

Reply:
> `sphinx-autobuild` is included in the poetry dependencies.

done.